docs/PowerAuth-Enrollment-Server-1.5.0.md

```diff
@@ -0,0 +1,35 @@
+# Migration from 1.4.x to 1.5.x
+
+This guide contains instructions for migration from PowerAuth Enrollment Server version `1.4.x` to version `1.5.0`.
+
+
+## Database Changes
+
+
+### Custom Data
+
+The type of column `es_onboarding_process.custom_data` has been changed.
+
+
+#### PostgreSQL
+
+```sql
+ALTER TABLE es_onboarding_process
+    ALTER COLUMN custom_data TYPE TEXT;
+```
+
+
+#### Oracle
+
+```sql
+ALTER TABLE es_onboarding_process
+    MODIFY custom_data CLOB;
```

Comment:
This modification does not work in Oracle, migration needed, see https://logic.edchen.org/how-to-modify-varchar2-to-clob-of-a-column/
@romanstrobl suggested to have a different column for FDS data.